health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js

```
@@ -7,356 +7,318 @@ import { Card } from "@egovernments/digit-ui-components";
+    if (customProps?.sessionData?.BOUNDARY?.boundarySelection?.selectedData) {
+      setSelectedData(customProps?.sessionData?.BOUNDARY?.boundarySelection?.selectedData);
+    }
+  }, [customProps?.sessionData?.BOUNDARY?.boundarySelection?.selectedData]);
+
+  // Memoizing parents and parent_group to only compute once when selectedData changes
+
+  const { parents, parent_group } = useMemo(() => {
+    if (!selectedData) {
+      return { parents: null, parent_group: null };
+    }
+    let parents = {};
```

Biome

[error] This let declares a variable that is only assigned once.

'parents' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)